### DISCUSSION TITLE
Is possible to see the current value of a slider field?

### DISCUSSION THREAD

**Discussion by botzill** [directus/directus#3005] [Archive]
When we edit a field with a slider and edit it

<img width="836" alt="Screen Shot 2020-11-05 at 21 28 13" src="https://user-images.githubusercontent.com/4686040/98287358-ebc08c80-1fad-11eb-842d-4ac1ac498f94.png">

is possible to view the current value by default without clicking on it and see it?

**Comment by benhaynes** [comment]
I pushed a fix for this... now there's an "Always Show Value" option for the interface. Also, I made it so the value is always shown on hover too.

Also, I cleaned up the styling quite a bit too. Now the hit/click area is bigger, the bar uses the primary color, the ticks/values are cleaner.

![Screen Shot 2020-11-06 at 10 08 38 AM](https://user-images.githubusercontent.com/522079/98381854-73db7c00-2018-11eb-8e04-6ed28a0f8787.png)
![Screen Shot 2020-11-06 at 10 09 16 AM](https://user-images.githubusercontent.com/522079/98381873-776f0300-2018-11eb-9fdc-d45b148cec32.png)

**Comment by botzill** [comment]
Perfect!
